packages/contract-tests/test/errors.js

```diff
@@ -236,14 +235,16 @@ describe("Client appends errors (vmErrorsOnRPCResponse)", function () {
       const example = await Example.new(1);
       try {
         const accounts = await Example.web3.eth.getAccounts();
-        await example.runsOutOfGas({ from: accounts[0], gas: "0x6691b7" });
+        // specifying a gasLimit as ganache @2.7.0 no longer limits gas
+        // estimations to the default block gas limit.
```

Comment:
Maybe either remove this comment or reword it to be independent of history (we can say _why_ we specify `gasLimit`... seems better than commenting on the past)